apps/sensenet/cypress/integration/content/create_delete_content.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+const folderName = `TestFolder`
+
+describe('Create/Delete content', () => {
+  before(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+  })
+  it('Creating a new folder should work properly', () => {
+    cy.get('[data-test="drawer-menu-item-content"]').click()
+    cy.get('[data-test="menu-item-it-workspace"]').click()
+    cy.get('[data-test="add-button"]').click()
+    cy.get('[data-test="listitem-folder"]')
+      .click()
+      .then(() => {
+        cy.get('#DisplayName').type(folderName)
+        cy.get('#Name').type(folderName)
+        cy.contains('Submit').click()
+        cy.get('[data-test="snackbar-close"]').click()
+        cy.get(`[data-test="table-cell-${folderName.replace(/\s+/g, '-').toLowerCase()}"]`).should(
+          'have.text',
+          folderName,
+        )
+      })
+  })
+
+  it('Folder delete should work properly', () => {
+    cy.get('[data-test="drawer-menu-item-content"]').click()
```

Comment:
check first if the folder exists, so the check will prove it is was there before the deletion

Reply:
if there is no folder with this data-test attribute then it will fail anyway